src/main/java/javax/jmdns/impl/DNSIncoming.java

```diff
@@ -301,12 +307,15 @@ private DNSRecord readAnswer(InetAddress source) {
         String domain = _messageInputStream.readName();
         DNSRecordType type = DNSRecordType.typeForIndex(_messageInputStream.readUnsignedShort());
         if (type == DNSRecordType.TYPE_IGNORE) {
-            logger.warn("Could not find record type. domain: {}\n{}", domain, this.print(true));
+            logger.warn("Could not find record type. domain: {}", domain);
```

Comment:
Please assign `_messageInputStream.readUnsignedShort()` to a variable (maybe `short wireType`) and log this value in the message to help identify the missing / unknown type.

Reply:
`DNSRecordType.typeForIndex()` logs this value on WARN level, so it has already been there in the log (similarly as above) . I would like to keep this fix as small as possible.

Reply:
Same as above.